src/items/traits.md

```diff
@@ -110,6 +115,8 @@ trait TraitMethods {
 trait NonDispatchable {
     // Non-methods cannot be dispatched.
     fn foo() where Self: Sized {}
+    // Requires unstable `unsized_fn_params` feature.
```

Comment:
The other reference to the unstable feature.

Reply:
Oops, fixed in https://github.com/rust-lang/reference/pull/1248/commits/bc9a036c1e00330400139db457c11d999ecbd673